modules/Attendance/Presenters/LiveTrackingPresenter.php

```
@@ -23,9 +23,19 @@
 
         return [
             'attendance_id' => $this->attendance->id,
-            'employee_name' => $this->attendance->user->name ?? 'N/A',
-            'branch_name'   => $this->attendance->user->branch->name ?? 'N/A',
-            // 'department_name' => 'إدارة الموارد البشرية',
+            'user' => $this->attendance->user ? [
+                'id' => $this->attendance->user->id ?? '-',
+                'name' => $this->attendance->user->name ?? '-',
+                'email' => $this->attendance->user->email ?? '-',
+                'phone' => $this->attendance->user->phone ?? '-',
+                'company_name' => $this->attendance->user->company->name ?? '-',
+                'birthdate' => $this->attendance->user->companyUser->birthdate_gregorian ?? '-',
+                'gender' => $this->attendance->user->companyUser->gender ?? '-',
+                'branch_name'   => $this->attendance->user->professionalData->branch->name ?? '-',
```

The professional data relationships (`professionalData->branch`, `professionalData->department`, `professionalData->management`) are deeply nested. This could fail silently if any intermediate relationship is null. Consider adding null checks or restructuring the data access.